src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -426,15 +425,22 @@ private boolean tryToLaunchJournalNode(SchedulerDriver driver, Offer offer) {
     } else if (deadJournalNodes.contains(offer.getHostname())) {
       launch = true;
     }
-    if (launch) {
-      return launchNode(
-          driver,
-          offer,
-          HDFSConstants.JOURNAL_NODE_ID,
-          Arrays.asList(HDFSConstants.JOURNAL_NODE_ID),
-          HDFSConstants.NODE_EXECUTOR_ID);
+    if (!launch) {
+      return false;
     }
-    return false;
+
+    String journalNodeTaskName = getNextTaskName(HDFSConstants.JOURNAL_NODE_ID);
+    if (journalNodeTaskName.isEmpty()) return false;
```

Comment:
To be consistent with our style here, we need to use brackets and put return false on the next line. We want to keep our style consistent throughout the project ideally.